src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.html

```diff
@@ -22,30 +22,35 @@ <h3 class="wrapper-title">ДОДАТИ ДОСЯГНЕННЯ</h3>
         </app-validation-hint>
 
         <label class="step-label">Учасники<span class="step-required">*</span></label>
-        <mat-form-field appearance="none">
-          <mat-select multiple disableOptionCentering formControlName="childrenIDs" 
-            panelClass="dropdown-panel" class="step-input">
-            <mat-select-trigger>
-              <mat-chip-list>
-                <mat-chip 
-                  *ngFor="let child of AchievementFormGroup.controls['childrenIDs'].value"
-                  [removable]="true" (removed)="onRemoveItem(child, 'childrenIDs')">
-                  {{child.lastName}} {{child.firstName}} {{child.middleName}}
-                  <mat-icon matChipRemove>cancel</mat-icon>
-                </mat-chip>
-              </mat-chip-list>
-            </mat-select-trigger>
-            <mat-option *ngFor="let child of children$"   
-              [value]="child" class="dropdown-option">
-              {{child.lastName}} {{child.firstName}} {{child.middleName}}
-            </mat-option>
-          </mat-select>
-        </mat-form-field>
-        <app-validation-hint 
-          [validationFormControl]="AchievementFormGroup.get('childrenIDs')"
-          [isTouched]="AchievementFormGroup.get('childrenIDs').touched"
-        >
-        </app-validation-hint>
+
+        <ng-container *ngIf="approvedChildren.totalAmount !== 0; else noChildren"> 
```

Comment:
*ngIf="approvedChildren.totalAmount;" (this is enough, because 0 return you false)